roles/swift-ring/handlers/main.yml

```diff
@@ -1,5 +1,19 @@
 ---
-- name: setup swift rings
+- name: setup swift rings - bootstrap
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
-           bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
+           --config {{ swift_ring.outdir }}/ring_definition.yml --outdir {{ swift_ring.outdir }} bootstrap
```

Comment:
I don't see where you're passing the `-u` to swift tool. The existing command is doing `sudo -u swiftops /usr/local/bin/swifttool <args>`

Sudo is getting the -u, not swifttool . What am I missing?